### PR TITLE
Allow passing of an image to LinkedData, and get the branded image

### LINKED ISSUE #7618: Schema.org image for CPS assets
**Is your feature request related to a problem? Please describe.**
At the moment, the Schema.org (linked data) image associated with a page is hardcoded to be the brand asset.  If the asset has an indexImage in the Ares feed, then we should use that instead.

(This work has already been done for the og:image, so we may be able to replicate that implementation)

**Describe the solution you'd like**
Update the logic for STY, MAP, and PGL assets so that the Schema.org image for the asset is the index image, where there is an index image in the Ares data (using the branded version of the ichef recipe, as we do for the og:image).  If there is no index image then we should use the brand image as at present.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] Validate the schema tags
- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #7618 

**Overall change:**
Allows the structured data (LinkedData) to take an imageLocator prop in order to override the image used for some page types

**Code changes:**

- Allow passing of new `imageLocator` prop to linkedData container
- if imageLocator prop is present use the `getBrandedImage` util to generate the branded image URL from the locator
- If branded image is present and not null use this as the image value for structured data, otherwise default to existing log image
- Ensures PGL, MAP & STY pages pass in the index image (if it exists) as the imageLocator prop

Google Rich Results Test (Structured Data Tests)
https://search.google.com/test/rich-results?id=mSpaVmMjYqx6jnCAh9rMCA

image

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by jroebu14 — APPROVED** [review]

File: src/integration/pages/storyPage/mundo/__snapshots__/canonical.test.js.snap [hunk 1]
```diff
@@ -306,7 +306,7 @@ Object {
       "image": Object {
         "@type": "ImageObject",
         "height": 576,
-        "url": "https://news.files.bbci.co.uk/ws/img/logos/og/mundo.png",
+        "url": "http://ichef.test.bbci.co.uk/news/1024/branded_mundo/17D42/production/_110620679_gettyimages-542984918.jpg",
```
Comment (simonsinclair) [inline src/integration/pages/storyPage/mundo/__snapshots__/canonical.test.js.snap]: I'm curious, why do these go from secure to non-secure?

**Review by simonsinclair — APPROVED** [review]

File: src/integration/pages/storyPage/mundo/__snapshots__/canonical.test.js.snap (hunk 1 above)
Comment (chris-hinds) [reply src/integration/pages/storyPage/mundo/__snapshots__/canonical.test.js.snap]: thats a good spot, wonder if it is just our localhost setup. will take a look.

Comment (chris-hinds) [reply src/integration/pages/storyPage/mundo/__snapshots__/canonical.test.js.snap]: This has been fixed now.

File: src/app/pages/StoryPage/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -3412,7 +3412,7 @@ exports[`Story Page should render correctly when the secondary column data is no
       property="og:description"
     />
     <meta
-      content="http://ichef.test.bbci.co.uk/news/1024/branded_pidgin/6ACE/test/_63724372_gettyimages-1098075358.jpg"
+      content="https://ichef.test.bbci.co.uk/news/1024/branded_pidgin/6ACE/test/_63724372_gettyimages-1098075358.jpg"
```
Comment (simonsinclair) [inline src/app/pages/StoryPage/__snapshots__/index.test.jsx.snap:3415]: 🎉

**Review by simonsinclair — APPROVED** [review]
This is a thorough bit of work. Nice one, Chris.

**Comment by paruchurisilpa** [comment]
I was able to check this using example Chris provided:

http://localhost:7080/amharic/news-53260525

"indexImage": {
            "altText": "ሃጫሉ",
            "copyrightHolder": "Family",
            "height": 549,
            "href": "http://c.files.bbci.co.uk/5042/production/_113164502_mediaitem113164501.jpg",
            "id": "113164502",
            "path": "/cpsprodpb/5042/production/_113164502_mediaitem113164501.jpg",
            "subType": "index",
            "type": "image",
            "width": 976
        },

For other assets which do not have indexImage in ares feed, brand image is used.

LGTM..

**Pull request merged**